packages/elements/src/combo-box/__test__/combo-box.template.test.js

```diff
@@ -8,19 +8,21 @@ describe('combo-box/Template', () => {
   describe('Template Parts', () => {
     it('Empty DOM has all required parts', async () => {
       const el = await fixture('<ef-combo-box lang="en"></ef-combo-box>');
-      expect(el).shadowDom.to.equalSnapshot(snapshotIgnore);
+      await expect(el).shadowDom.to.equalSnapshot(snapshotIgnore);
     });
 
     it('Placeholder is rendered', async () => {
       const el = await fixture('<ef-combo-box placeholder="Placeholder" lang="en"></ef-combo-box>');
-      expect(el).shadowDom.to.equalSnapshot(snapshotIgnore);
-      expect(el.placeholder).to.equal('Placeholder', 'Placeholder is not reflected to attribute');
-      el.setAttribute('placeholder', 'New Placeholder');
+      expect(el.placeholder).to.equal('Placeholder', 'Incorrect placeholder value');
```

Comment:
This does not make sure that placeholder attribute is pass onto `input` and rendered.

Reply:
I already splitted this to 2 test cases